web/docs/guides/with-flutter.mdx

```diff
@@ -628,7 +640,276 @@ flutter run -d web-server --web-hostname localhost --web-port 3000
 
 And then open the browser to [localhost:3000](http://localhost:3000) and you should see the completed app.
 
-<img height="500px" src="/img/supabase-flutter-demo.png" alt="Supabase User Management example" />
+<img src="/img/supabase-flutter-account-page.png" alt="Supabase User Management example" />
+
+
+## Bonus: Profile photos
+
+Every Supabase project is configured with [Storage](/docs/guides/storage) for managing large files like 
+photos and videos.
+
+### Adding image uploading feature to Account page
+
+We will use [`image_picker`](https://pub.dev/packages/image_picker) plugin to select an image form the device.
+
+Run the following command to install it.
+
+```bash
+flutter pub add image_picker
+```
+
+Using [`image_picker`]((https://pub.dev/packages/image_picker)) requires some additional preparation depending on the platform. 
```

Comment:
I think this one need one set of brackets removed - `[](( ))` should be `[ ]( )`

Reply:
@kiwicopple 
Ah, thanks! Fixed!